Meta/run.sh

```diff
@@ -387,14 +389,6 @@ elif [ "$SERENITY_RUN" = "ci" ]; then
         -append "${SERENITY_KERNEL_CMDLINE}"
 else
     # Meta/run.sh: qemu with user networking
-    if [ "$SERENITY_ARCH" = "aarch64" ]; then
```

Comment:
this is temporary and will be reverted before the switching the PR to non-draft FYI.